.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: Continuous Integration 
+
+on:
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install dependencies
+        run: pip install -e ".[dev]"
+      - name: Run Tox
+        run: tox -e py
```

Comment:
```suggestion
        run: tox -e py

```

Reply:
To get rid of that missing newline warning

Reply:
Done.